packages/beacon-node/src/chain/lightClient/index.ts

```diff
@@ -274,65 +276,27 @@ export class LightClientServer {
    * - Has the most bits
    * - Signed header at the oldest slot
    */
-  async getUpdates(period: SyncPeriod): Promise<altair.LightClientUpdate> {
```

Comment:
Leave this function as is, and move the code

```ts
    const periods: number[] = Array.from({length: count}, (_ignored, i) => i + startPeriod);
    return await Promise.all(periods.map((period) => this.doGetUpdate(period)));
```

this is very opinionated and each consumer should decide how to handle it

Reply:
I think perhaps the chain lightclient module should not even have a `getUpdates` but rather a `getUpdate` since the return type is just a singular `LightClientUpdate`.

Reply:
i would find it useful to req/resp multi updates to get to the current period fast :thinking:

Reply:
> I think perhaps the chain lightclient module should not even have a `getUpdates` but rather a `getUpdate` since the return type is just a singular `LightClientUpdate`.

Do not forget that the lightclient module also serves this rest endpoint, wherein, if the return type is specified as json, it does require `LightClientUpdate[]` to be returned.